myapp/templates/base.html

```diff
@@ -15,6 +15,7 @@
     <link rel="stylesheet" href="{% static 'css/font-awesome.min.css' %}">
     <link href="{% static 'css/addons/datatables.min.css' %}" rel="stylesheet">
     <link href="{% static 'css/main.css' %}" rel="stylesheet" type="text/css">
+      <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
Not sure why you have to import another icon font; Font Awesome 'fa fa-' works fine

Reply:
good to know